src/src.pro

```diff
@@ -435,7 +435,7 @@ else:linux:!android {
     INSTALLS += dbus_service
 
     systemd_service.files = ../linux/debian/mozillavpn.service
-    systemd_service.path = /lib/systemd/system
+    systemd_service.path = /usr/lib/systemd/system
```

Comment:
Do we need to include this change in 2.3?

Reply:
This install target only exists in the qmake project to satisfy users who want to install from sources, and on Debian-based distributions it makes no difference since /usr/lib/systemd and /lib/systemd are hardlinked to the same place.